src/richie/apps/core/templates/richie/base.html

```diff
@@ -126,7 +126,9 @@
                         </a>
                     </div>
                     <div class="body-footer__insert">
-                        <p class="body-footer__title">{% trans "Life-changing learning!" %}</p>
+                        {% block body_footer_slogan %}
+                            <p class="body-footer__title">{% trans "Life-changing learning!" %}</p>
+                        {% endblock body_footer_slogan %}
```

Comment:
Well for coherence, since it is called title:
```suggestion
                        {% block body_footer_title %}
                            <p class="body-footer__title">{% trans "Life-changing learning!" %}</p>
                        {% endblock body_footer_title %}
```